docs/pair-mob-programming.md

```diff
@@ -0,0 +1,109 @@
+---
+id: pair-mob-programming
+title: Desarrollo de Software en Parejas o Grupos
+sidebar_label: Desarrollo de Software en Parejas o Grupos
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+El desarrollo de un software nuevo no es un proceso sencillo, dependiendo del tamaño del programa, se deberá tener en cuenta una gran cantidad de posibles coyunturas, funciones y cuestiones problemáticas.  
+De ahí que en estos últimos años se hayan ido desarrollando otros métodos de trabajo más modernos que permiten programar con mayor eficiencia y generar un código libre de errores.
+
+
+## Pair Programming
+
+### ¿Qué es Pair programming?
+El pair programming específica que siempre haya dos personas trabajando al mismo tiempo en el código y que, en la medida de lo posible, se sienten juntas. Una se encarga de escribir el código y la otra de supervisarlo en tiempo real. Al mismo tiempo, están constantemente intercambiando impresiones: debaten problemas, encuentran soluciones y desarrollan ideas creativas. 
+
+<div style={{  textAlign: "center" }}>
+<img alt="Imagen Pair Programing"  src={ useBaseUrl( 'img/metodosProgramacion/PairProgramming.png' )} /> </div>
+
+### ¿Cómo se hace?
+**Tamaño del team:** 2 personas.
+
+### Roles:
+- **Conductor:** Se encarga de escribir el código. 
+- **Navegador:** Se encarga de supervisar ese código. 
+
+Una de las reglas del pair programming establece que estos dos roles se intercambien con regularidad, de esta manera se evita una posible brecha jerárquica.
+
+### Tipos de pair programming:
+- **Backset Navigator:** El navegador asume más de los roles tácticos del conductor. En la navegación por asiento trasero, el controlador sigue controlando el teclado y escribiendo, pero el navegador dicta instrucciones sintácticas, como qué nombre llamar a una variable o qué método específico llamar. El estilo de navegador del asiento trasero funciona mejor con un principiante como conductor y experto como el navegante, permitiendo que el principiante aprenda haciendo.
```

Comment:
navegador

Reply:
Backseat Navigator -> Backseat Navegador? 
OR
Backseat Navigator -> Navegador?

Reply:
I would leave it as  `Navegador`

The reason i mentioned `backseat` is because it was misspelled `backset`